### ISSUE TITLE
`walltime` is not always applied.

### ISSUE THREAD

**Issue by forsyth2** [E3SM-Project/zppy#273]
`walltime` appears to not always be applied -- see discussion on #248.

**Comment by BunnyVon** [comment]
I tried your configuration for a smaller job on cori. Two issues occurred: 1) walltime doesn't change to 30min; 2) the jobs still use "regular" queue instead of "debug" queue.

**Comment by forsyth2** [comment]
@BunnyVon You have both `qos = debug` and `walltime = "00:30:00"` set in your `cfg`? If so, it's very strange that neither of those are being picked up.

**Comment by BunnyVon** [comment]
Yes, I have both set for the cfg file. btw, I'm using e3sm_unified environment. I saw somewhere you have your own conda env. would that cause issues?

**Comment by forsyth2** [comment]
Hmm I don't think that would cause any issues, I'm not sure what's going on.
